concepts/generics/links.json

```diff
@@ -0,0 +1,6 @@
+[
+  {
+    "url": "https://elmprogramming.com/type-system.html",
+    "description": "Elm type system"
+  }
```

Comment:
We can add the ones from the hints.

Reply:
I don't think the links in the hints would make sense here. Can you give them another look?

Reply:
Specifically I was thinking of https://guide.elm-lang.org/types/custom_types.html, I think it fits quite well. You are right the other don't quite fit.